gradle/libs.versions.toml

```diff
@@ -26,6 +26,12 @@ powerMock-moduleJunit4 = { module = "org.powermock:powermock-module-junit4", ver
 reflections = "org.reflections:reflections:0.10.2"
 scala-library = "org.scala-lang:scala-library:2.13.8"
 slf4j-lambda = "kr.pe.kwonnam.slf4j-lambda:slf4j-lambda-core:0.1"
+# @pin because slf4j-simple 2.0.0 isn't compatible with slf4j-api v1.x.y, which is still pulled in by our dependencies.
```

Comment:
Probably one of the longest-awaited releases in the JVM ecosystem, ever 😅
Basically:
* adds lambda API: `log.info(() -> "lolxd")` (we still can't use it tho, as long as IntelliJ itself isn't migrated to slf4j v2.0.0)
* adds fluent API: `log.atInfo().log("lolxd")`
* changes detection of implementation from the somewhat primitive-yet-effective static binding, to service provider (that's a breaking change)